src/components/FileVirtualGrid.vue

```diff
@@ -0,0 +1,143 @@
+
+<template>
+	<a :class="{'file--clear': !loaded}"
```

Comment:
Why do we need a new dedicated template?
Ideally we should keep using the same File component that any other Views. That way any change is visible and propagated to all! :thinking:

Reply:
Because props are passed from the lib and as I didn't want to change the other views Grid system i created a new one, but i can remove it for sure